source/_includes/installation/core.md

```diff
@@ -33,7 +33,7 @@ sudo apt-get upgrade -y
 Install the dependencies:
 
 ```bash
-sudo apt-get install -y python3 python3-dev python3-venv python3-pip bluez libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata
+sudo apt-get install -y python3 python3-dev python3-venv python3-pip bluez libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata ffmpeg liblapack3 liblapack-dev libatlas-base-dev
```

Comment:
`ffmpeg` sounds fine to add, I guess. The more default config will indeed trigger the need for it quickly.

I'm unsure about lapack & atlas (and the dev headers), what dependency would need that?

../Frenck

Reply:
> able to import component: liblapack.so.3: cannot open shared object file: No such file or directory
> 2023-03-19 03:34:34.723 ERROR (MainThread) [homeassistant.setup] Setup failed for mobile_app: Unable to import component: liblapack.so.3: cannot open shared object file: No such file or directory

So that means we need the libs, not the development headers.

Reply:
> 

if you would be right you wouldn't need other dev packages in the dependencies too